File: DSA 4th Year/TUF/Patterns/number_crown.py
n = 3
for i in range(1, n+1):
    for j in range(1, n+1):
        if j <= i:
            print(j, end=" ")
        else:
            print(" ", end=" ")
    spaces = n-i
    while (spaces != 0):
        print(" ", end=" ")
        spaces -= 1
    for k in range(i, 0, -1):
        print(k, end=" ")

    print()

File: DSA 4th Year/TUF/Patterns/number_crown.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{

    int n = 13;
    for (int i = 1; i <= n; i++)
    {
        for (int j = 1; j <= n; j++)
        {
            if (j <= i)
            {
                cout << j<<" ";
            }
            else
            {
                cout << "  ";
            }
        }
        // print the spaces
        int spaces = n - i;
        while (spaces != 0)
        {
            cout << "  ";
            spaces--;
        }
        for (int k = i; k >= 1; k--)
        {
            cout << k<<" ";
        }

        cout << endl;
    }

    return 0;
}